.github/workflows/plot-tests.yml

```diff
@@ -1,31 +1,41 @@
-  - name: Set up Python 3.11
-    uses: actions/setup-python@v5
-    with:
-      python-version: '3.11'
+name: Mathics3 (ubuntu)
 
-  - name: Install OS dependencies
-    run: |
-      sudo apt-get update -qq
-      sudo apt-get install -qq liblapack-dev llvm-dev tesseract-ocr remake
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '**'
```

Comment:
Sorry, not familiar with workflows - does this mean it runs on pull requests on any branch? Just checking that the intention is in general this should run on pull requests.

Reply:
I copy and paste from other workflow, and I forgot changing the name. Thanks for noticing it!

Reply:
The workflow is executed on each PR.